c2corg_ui/static/js/maincontroller.js

```diff
@@ -69,3 +69,10 @@ app.MainController.prototype.isPath = function(path) {
   }
 };
 
+
+/**
+ * @export
+ */
+app.MainController.prototype.animateHeaderIcon = function(e) {
+  app.utils.animateHeaderIcon(e);
```

Comment:
I tried but it doesn't seem to work, as if it was awaiting a controller function. But I find that 'main controller' is supposed to represent a controller that can be used application-wide, and in this case we may want to use this particular function in multiple parts of the app.